### DISCUSSION TITLE
Obsidian Vault 状态大盘工具

### DISCUSSION THREAD

**Discussion by giscus** [edony-ink/comments#26] [General]
# Obsidian Vault 状态大盘工具

借助 Dataview、Obsidian Chart 对 Better Word Count 插件中的 Obsidian Vault 状态统计数据进行可视化展示，实现的关键是一段 dataviewjs 脚本，参考文中的 gist 链接。

https://www.edony.ink/chartjs-tools-for-obsidian-vault/

**Comment by edonyzpc** [comment]
数据源自插件 better word count，插件数据更新的逻辑跟 dashboard 又不匹配的地方，例如如果没有更新数据会变成0，可以考虑通过自己的插件 personal assistant 来搭配使用
